packages/app/src/app/pages/Sandbox/Editor/Header/ForkButton/ForkButton.tsx

```diff
@@ -153,64 +166,79 @@ export const ForkButton: React.FC<ForkButtonProps> = props => {
         </Button>
       )}
       {otherWorkspaces.length ? (
-        <Menu>
-          <Menu.Button
-            variant={props.variant}
-            css={
-              state.activeWorkspaceAuthorization === 'READ'
-                ? {}
-                : {
-                    width: '26px',
-                    borderTopLeftRadius: 0,
-                    borderBottomLeftRadius: 0,
-                  }
-            }
-          >
-            {state.activeWorkspaceAuthorization === 'READ' ? (
-              <>
-                <ForkIcon css={css({ height: 3, marginRight: 1 })} />{' '}
-                <Text css={css({ marginRight: 2 })}>Fork</Text>
-              </>
-            ) : null}
-            <Icon size={8} name="caret" />
-          </Menu.Button>
-          <Menu.List
-            css={css({
-              fontSize: 2,
-              lineHeight: 1,
-            })}
-            style={{
-              paddingTop: 4,
-              paddingBottom: 4,
-              marginLeft: '-4rem',
-              marginTop: -4,
-            }}
-          >
-            {currentSpace && (
-              <TeamOrUserItem
-                forkClicked={props.forkClicked}
-                item={currentSpace}
-                disabled={state.activeWorkspaceAuthorization === 'READ'}
-                isPersonal={currentSpace.teamId === state.personalWorkspaceId}
-              />
-            )}
-
-            <Menu.Divider />
-            {otherWorkspaces.map((space, i) => (
-              <TeamOrUserItem
-                isPersonal={space.teamId === state.personalWorkspaceId}
-                key={space.teamId}
-                forkClicked={props.forkClicked}
-                item={space}
-                disabled={
-                  space.userAuthorizations.find(
-                    authorization => authorization.userId === user.id
-                  )?.authorization === 'READ'
+        <Tooltip
+          label={
+            preventForksOutsideWorkspace
+              ? 'You don not have permission to fork this sandbox outside of the workspace'
+              : null
```

Comment:
Does nothing break when the label is null?

Reply:
The opposite :)

If it's null, then the Tooltip renders its children without adding a tooltip. 

Would love to revisit some components to make this API more explicit